src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateXprojProjectReferencesRule.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+    // TODO: XProj ProjectToProject references
+    public class MigrateXprojProjectReferencesRule : IMigrationRule
+    {
+        public void Apply(MigrationSettings migrationSettings, MigrationRuleInputs migrationRuleInputs)
+        {
+            throw new NotImplementedException();
```

Comment:
How about the comment above moves to the message: `// TODO: XProj ProjectToProject references`